contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,382 @@
+---
```

Comment:
This file is a rework of this existing one -> https://posthog.com/docs/getting-started/identify-users

I've stripped out a lot, in the hopes of making things clearer and easier to follow. Let me know if I've missed anything important